README.md

```diff
@@ -1,14 +1,45 @@
-# Project
+# Ingesting Data from Splunk to Azure Data Explorer
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+In this repository there are following ways to ingest data from Splunk to Azure Data Explorer.
```

Comment:
The project provides modules to integrate Azure Data Explorer (on Azure and Microsoft Fabric) with Splunk

a) Export data continuously from Splunk index to ADX (Interval based polling and export)
b) Export data to ADX as a target using [Splunk universal forwarder](https://www.splunk.com/en_us/download/universal-forwarder.html)
c) A sample [Spark](https://spark.apache.org/) job to extract existing data from Splunk indexes to ADX for historical analysis